hack/otel/compose.yaml

```diff
@@ -0,0 +1,27 @@
+name: moby-otel
+
+services:
+
+  jaeger:
+    image: jaegertracing/all-in-one:latest
```

Comment:
I think it's worth adding `restart: always` to every service.

Reply:
Good idea. Reading the compose spec i see `always: The policy always restarts the container until its removal.`. 
Maybe using `unless-stopped` might be better in this case? I wouldn't want potential users (even if we kinda only expect devs and power-users to use this) to accidentally have these services running forever

Reply:
I thought `unless-stopped` wouldn't be right here because AFAIK if the container is stopped during dockerd shutdown, it won't automatically restart (eg. on a reboot). But I tried it and it seems to work properly, so we can also consider using that restart policy.